--- /etc/rc.subr.orig	2021-04-04 09:46:40.096987851 +0000
+++ /etc/rc.subr	2021-04-04 11:10:05.841203420 +0000
@@ -514,9 +514,12 @@
 			eval ${rcvar}=YES
 		fi
 		;;
+	enabled)
+		_rc_quiet=yes
+		;;
 	esac
 
-	_keywords="start stop restart rcvar"
+	_keywords="enabled start stop restart rcvar"
 	if [ -n "$extra_commands" ]; then
 		_keywords="${_keywords} ${extra_commands}"
 	fi
@@ -571,11 +574,13 @@
 			fi
 			for _elem in $_keywords; do
 				if [ "$_elem" = "$rc_arg" ]; then
-					cat 1>&2 <<EOF
+					if [ -z "$_rc_quiet" ]; then
+						cat 1>&2 <<EOF
 \$${rcvar} is not enabled - see ${rcvar_manpage}.
 Use the following if you wish to perform the operation:
   $0 one${rc_arg}
 EOF
+					fi
 					exit 1
 				fi
 			done
@@ -617,6 +622,11 @@
 
 		case "$rc_arg" in	# default operations...
 
+		enabled)
+			# if we made it here, we are enabled
+			exit 0
+			;;
+
 		status)
 			if [ -n "$rc_pid" ]; then
 				echo "${name} is running as pid $rc_pid."
